aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/servlet/AwsHttpServletRequest.java

```diff
@@ -280,6 +280,7 @@ protected String generateQueryString(Map<String, String> parameters) {
         if (headerValue == null) {
             return values;
         }
+        int entryCounter = 1;
```

Comment:
Not used, forgot there from when I was using the counter to name the key if there was no other name specified. Good catch. Removed.